From afb5717e67727d49fae0d2a1a4169e5b247387f4 Mon Sep 17 00:00:00 2001
From: Andrew Halaney <ahalaney@redhat.com>
Date: Mon, 5 Jun 2023 12:16:24 -0500
Subject: [PATCH] fix(kernel-modules): add interconnect drivers

Many devicetree based systems require interconnect drivers to function
properly. Include those in the initramfs.

Signed-off-by: Andrew Halaney <ahalaney@redhat.com>
---
 modules.d/90kernel-modules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index 5b6df776a9..e217512594 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -71,6 +71,7 @@ installkernel() {
                 "=drivers/gpio" \
                 "=drivers/hwmon" \
                 "=drivers/hwspinlock" \
+                "=drivers/interconnect" \
                 "=drivers/i2c/busses" \
                 "=drivers/mailbox" \
                 "=drivers/memory" \
